genie-docs/src/docs/asciidoc/_releaseNotes.adoc

```diff
@@ -50,6 +50,11 @@ See https://cloud.spring.io/spring-cloud-aws/[Spring Cloud AWS] for more informa
 See https://cloud.spring.io/spring-cloud-aws/[Spring Cloud AWS] for more information.
 |false
 
+|management.endpoints.web.base-path
```

Comment:
Oh, i see it's something you do abtually... (copy properties doc into release notes)